_inc/client/security/index.jsx

```diff
@@ -83,6 +86,25 @@ export class Security extends Component {
 			return null;
 		}
 
+		const planClass = getPlanClass( get( this.props.sitePlan, [ 'product_slug' ] ) );
+		const activePurchaseProductSlugs = this.props.activeSitePurchases.map(
+			purchase => purchase.product_slug
+		);
+
+		const isPersonalPlan = 'is-personal-plan' === planClass;
+		const isFreePlanWithBackup =
```

Comment:
These pieces of logic will likely be useful for helpers in other places of the code, so it would be useful to extract them as separate helper functions, but that's totally non-mandatory for now, and can be done in another PR.

Reply:
Created https://github.com/Automattic/jetpack/issues/14114